python/lsst/cell_coadds/_exploded_coadd.py

```diff
@@ -115,10 +115,10 @@ def _make_view(original: ImageLike) -> ImageLike:
             yield ViewImagePlanes(cell.outer, _make_view, bbox=new_bbox)
 
     @property
-    def psf_image(self) -> ImageD:
+    def psf_image(self) -> ImageF:
         """A stitched-together image of the PSF models for each cell."""
         if self._psf_image is None:
-            stitched_psf_image = ImageD(self.psf_grid.bbox)
+            stitched_psf_image = ImageF(self.psf_grid.bbox)
```

Comment:
I don't think we need double precision for storing PSF images (it may be useful for some operations, including PSF estimation itself, but that's different from storage), and the use of double in the `Psf` base class is something I now regard as a hard-to-fix mistake.  I am hoping to distance new code that works on `MultipleCellCoadd` directly (instead of a stitched `ExposureF` view) from that mistake, and I also think the PSF images might be a large enough fraction of the total storage/memory cost of these things (especially for small cells) that it's worth being a bit stingy with bits.

Reply:
I might be misremembering, but I thought the PSF had to be double-precision or Erin was seeing biases with PSF deconvolution in metacal otherwise.

Reply:
Ah, my bad then.